src/EPPlus/Utils/ConvertUtil.cs

```diff
@@ -206,6 +206,52 @@ internal static double GetValueDouble(object v, bool ignoreBool = false, bool re
             }
             return d;
         }
+
+        /// <summary>
+        /// Convert an object value to a decimal 
+        /// </summary>
+        /// <param name="v"></param>
+        /// <param name="ignoreBool"></param>
+        /// <returns></returns>
+        internal static decimal GetValueDecimal(object v, bool ignoreBool = false)
```

Comment:
What if we take out one level of forking like this to reduce the code complexity:

```c#
internal static decimal GetValueDecimal(object value, bool ignoreBool = false)
{
    try
    {
        if (ignoreBool && value is bool)
        {
            return 0;
        }

        if (!IsNumericOrDate(value))
        {
            return 0;
        }

        if (value is DateTime dateTimeValue)
        {
            return Convert.ToDecimal(dateTimeValue.ToOADate());
        }

        if (value is TimeSpan timeSpanValue)
        {
            return Convert.ToDecimal(DateTime.FromOADate(0).Add(timeSpanValue).ToOADate());
        }

        return Convert.ToDecimal(value, CultureInfo.InvariantCulture);
    }
    catch
    {
        return 0;
    }
}
```